configure.ac

```diff
@@ -396,6 +399,35 @@ AS_IF([test "x$enable_shared" = "xyes"], [
     #  PowerPC and SPARC.
     # Therfore use -fPIC for now
     CXXFLAGS="$CXXFLAGS -fPIC"
+    # for make.config
+    DEFLIB1='$(LIB_SO)'
+    # for src/makefile
+    DEFLIB2+=' ../lib/libbout++.so'
```

Comment:
It would be good to use more descriptive names here, but it's tricky to think of good ones. `LIB_TO_BUILD` or something? I'm not sure

Reply:
the issue is I need to variables, that describe the default_lib / lib_to_build.
But `LIB_TO_BUILD{1,2}` would be better.